docs/core/testing/unit-testing-with-dotnet-test.md

```diff
@@ -144,21 +112,50 @@ at the root of `project.json`:
 }
 ```
 
-Finally, you need to set the framework node to use
+The template also sets the framework node to use
 `netcoreapp1.0`, and include the required imports to
-get xUnit.net to work with RC2:
+get xUnit.net to work with .NET Core RTM:
 
 ```json
   "frameworks": {
     "netcoreapp1.0": {
       "imports": [
-        "dnxcore50",
+        "dnxcore54",
```

Comment:
If imports are needed in this specific case, then shouldn't this change to `dotnet54`?  I'm not aware of a `dnxcore54`.